airflow/include/split.py

```diff
@@ -0,0 +1,64 @@
+import pandas as pd
+from typing import List
+
+from airflow.decorators import task
+
+from langchain.text_splitter import (
+    RecursiveCharacterTextSplitter,
+    Language,
+)
+from langchain.schema import Document
+
+# @task
+def split_markdown(df:pd.DataFrame):
```

Comment:
I think we should introduce tools like ruff and black to this project before we spend time on reviewing style, error that can be found/fixed by these tools